modules/kronatools/ktimporttaxonomy/main.nf

```diff
@@ -0,0 +1,39 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName; getProcessName } from './functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+process KRONATOOLS_KTIMPORTTAXONOMY {
+    tag "${meta.id}"
+    label 'process_high'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:meta, publish_by_meta:['id']) }
+
+    conda (params.enable_conda ? "bioconda::krona=2.8" : null)
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+        container "https://depot.galaxyproject.org/singularity/krona:2.8--pl5262hdfd78af_2"
+    } else {
+        container "quay.io/biocontainers/krona:2.8--pl5262hdfd78af_2"
+    }
+
+    input:
+    tuple val(meta), path(report)
+    path  "taxonomy/taxonomy.tab"
```

Comment:
I assume you are using this syntax so that the database is always made available to `kronatools` with this name?